### PR TITLE
Bootstrap angular app

### PR DESCRIPTION
**Pull request by ndsagaga**
1. Added all the app-component files (ts,html,css,spec)
2. Cleaned up code all unnecessary code in app-component (updated the test also)
3. Replaced prefix from app to scm (Following Angular style guidelines)

### CODE REVIEW

File: web/scm/src/styles.scss [hunk 1]
```diff
@@ -1 +1,3 @@
-/* You can add global styles to this file, and also import other style files */
+body {
+    margin: 0;
+}
```
Comment (dpronin) [inline web/scm/src/styles.scss:3]: we don't have this check here, but it's widely used across google -> Having blank new line at the end of the all text files

File: web/scm/src/app/app-component/app.component.spec.ts [hunk 2]
```diff
@@ -26,10 +26,10 @@ describe('AppComponent', () => {
     expect(app.title).toEqual('scm');
   });
 
-  it('should render title', () => {
+  it('should render para', () => {
```
Comment (dpronin) [inline web/scm/src/app/app-component/app.component.spec.ts:29]: Can we have test name like: 'Should display hello message' ? I could't found a doc for this right now, but it's something like this.

File: web/scm/src/styles.scss (hunk 1 above)
Comment (dpronin) [reply web/scm/src/styles.scss:3]: Also PTAL here https://github.com/google/gts

Comment (ndsagaga) [reply web/scm/src/styles.scss:3]: Got it. Will follow the style guide

File: web/scm/src/app/app-component/app.component.spec.ts (hunk 2 above)
Comment (ndsagaga) [reply web/scm/src/app/app-component/app.component.spec.ts:29]: Okay, will change that

**Pull request merged**